subjects/devops/play-with-containers/audit/README.md

```diff
@@ -2,30 +2,29 @@
 
 ##### Check the Repo content
 
-A `README.md` file and all files used to create and delete and manage the student infrastructure must be submitted in the repo.
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
 
 ###### Are all the required files present?
 
-###### `.env` file is not exists in the repo?
+###### `.env` file do not exist in the repo?
 
 ###### Are all pushed files in the repo clean of any credentials or passwords?
 
-##### Ask the following questions to the group or student
+##### Ask the following questions to the group or student.
 
-###### What are containers and what are their benefits of it?
+##### What are containers and what are their advantages?
 
-###### What is the difference between containers and virtual machines?
+##### What is the difference between containers and virtual machines?
 
-###### What is docker and what is its main role of it?
+##### What is Docker and what is it used for?
 
 ###### Did the student reply correctly to the questions?
 
-##### Check the Student Documentation
-The Documentation must exists in the `README.md` file!
+##### Open and read the `README.md` file provided by the student.
 
-###### Did the README.md file contains the all required information about the solution(Prerequisites, Configuration, Setup, Usage, ...)?
+###### Did the `README.md` file contains all the required information to run and manage the solution (prerequisites, configuration, setup, usage, etc)?

```

Comment:
Does ... contain, or if you want to keep it past just fix the s in contains, but since before it was used present is better to keep it consistent.

Reply:
done